build.gradle

```diff
@@ -967,6 +967,24 @@ project(':iceberg-pig') {
   }
 }
 
+project(':iceberg-aws') {
+  dependencies {
+    compile project(':iceberg-api')
+    compile project(':iceberg-bundled-guava')
+    compile project(':iceberg-core')
+    compile 'software.amazon.awssdk:glue'
+    compile 'software.amazon.awssdk:s3'
```

Comment:
Why does this pull in S3? Is it needed?

Reply:
I was thinking about the S3FileIO PR and marked it as compile dependency, let me change it for now to `testCompile`